pulpcore/cli/python/context.py

```diff
@@ -71,6 +71,7 @@ class PulpPythonRepositoryContext(PulpRepositoryContext):
     ENTITIES = _("python repositories")
     ID_PREFIX = "repositories_python_python"
     VERSION_CONTEXT = PulpPythonRepositoryVersionContext
+    CAPABILITIES = {"pulpexport": [PluginRequirement("python")]}
```

Comment:
This needs to include the version of pulp_python to include this feature.

Reply:
Ok, we have to wait for https://github.com/pulp/pulp_python/pull/580 to be included in the next pulp_python release.